### PR TITLE
NEPT-1452: Theme cleanup.

### PR DESCRIPTION
**Pull request by drupol**
This PR is linked to:

- Theme EC Europa: https://github.com/ec-europa/ec_europa/pull/15
- Nexteuropa: https://github.com/ec-europa/platform-dev/pull/1540
- Theme EC Resp: https://github.com/ec-europa/ec_resp/pull/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,20 @@ To create a new theme definition:
...
   automatically loaded.
   
+Atomium provides a custom page available on the path: **atomium-overview**.
+This particular page is only available to users with _administer themes_ permission.
+
+This page act as a showcase page of components.
+To add a component in there, your component needs to define one or two hooks:
+ - hook_atomium_definition_hook().
+   
+   This hook allows you to define simply a component.
```
Comment (voidtek) [inline README.md:125]: We can add here the option 
'disable' => TRUE, ?

Comment (drupol) [reply README.md:125]: I don't think it's necessary as I describe this in the .api.php file.

Comment (voidtek) [reply README.md:125]: Ok, it's true.

**Review by voidtek — APPROVED** [review]
This is fine for me.

**Comment by imanoleguskiza** [comment]
👍

**Pull request merged**